src/frontend/src/components/RecordSearch/Record/Case.tsx

```diff
@@ -19,6 +19,17 @@ export default class Cases extends React.Component<Props> {
       location,
       current_status,
     } = this.props.case;
+    let ineligibleCharges = 0; 
+    let allIneligible = 0
+    for (let i = 0; i < charges.length; i++){ 
+      let typeEligibility = charges[i].expungement_result.type_eligibility.status
+      if (typeEligibility == "Ineligible") {
+        ineligibleCharges++
+      }
+    }
+    if (ineligibleCharges == charges.length) {
+        allIneligible=1
+      }
```

Comment:
```suggestion
    let allIneligible = charges.every(charge => charge.expungement_result.type_eligibility.status === "Ineligible");
```

Note I have not tested this.